stock_average_daily_sale/models/stock_warehouse.py

```diff
@@ -29,11 +28,11 @@ def _compute_average_daily_sale_root_location_id(self):
                 continue
             warehouse.average_daily_sale_root_location_id = warehouse.lot_stock_id
 
-    @api.model_create_multi
```

Comment:
why changing this?

Reply:
As discussed at the meeting: it reflects differences between 16.0 and 14.0 in the Odoo standard stock_warehouse create override.